include/wifi_helper.h

```diff
@@ -0,0 +1,36 @@
+/* Sockets Example
+ * Copyright (c) 2020 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mbed.h"
+
+static const char *get_security_string(nsapi_security_t sec)
```

Comment:
this is just moved from the main class

Reply:
What is the incentive behind that move ?

Reply:
make examploe shorter, this is just a way to pretty print